docs/terminal/sydney.md

```diff
@@ -21,8 +21,10 @@
 * [Additional requirements](#airspace-structural-arrangements) must be met prior to opening SRI as a stand-alone position.
 
 ## Airspace
-The Sydney TMA spans a 45nm Radius around SY DME from `SFC` to `F280`. SY TCU is responsible for the Sydney TMA, except:  
+The Vertical limits of the SY TCU are `SFC` to `F280`.  
+SY TCU is responsible for the Sydney TMA, except:  
 
+- SY CTR `SFC` to `A005`  
```

Comment:
From memory, I think we designated SY TWR airspace as most but not all of the CTR.  Might need to specify that here

Reply:
fixed